Esri/arcgis-maps-sdk-swift-samples
# [New] Change map view background

## Description

This PR implements `Change mapview background` in `Map` category.
URL to README: https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/91f53720a587fb184b6e22fe171c9d4fdca75e19/Shared/Samples/Change%20map%20view%20background/README.md

Old sample: https://github.com/Esri/arcgis-runtime-samples-ios/tree/main/arcgis-ios-sdk-samples/Maps/Change%20map%20view%20background

## Linked Issue(s)

- `swift/issues/3581`

## How To Test

Tap "Background Grid Settings" button to display settings sheet
Change properties: "Color", "Line Color", "Line Width", "Size" and ensure the background grid in the MapView reflects those changes.

## To Discuss

- This sample does not have a common design, so the design is taken from the old sample viewer
